### PR TITLE
feature/restaurant-venue-cards

### PR DESCRIPTION
**Pull request by CamiloGilFranco**
## Summary

- Data is consumed from the backend for HomePage cards

- In restaurant cards and restaurant banners, the restaurant's real rating is rounded to show only one decimal place

- In the restaurant view, the Recommended component is exchanged for PopularRestaurant

- The path followed by the elements of the HomePage carousel is corrected

- The Location option is removed from the restaurant view

- The OVERVIEW option is replaced by VENUES in the restaurant view

- A card is created for each venue that shows image, name and city

- A modal is created that is displayed when clicking on the venues card and shows the complete information of each venue

### CODE REVIEW

File: src/assets/languages/languageSelector.js [hunk 1]
```diff
@@ -343,7 +346,7 @@ const languageSelector = (lang, str) => {
     filterSchedule: "Lunes a Viernes",
     cardDishesFrom: "Platos desde",
     restaurantOptionsOnline: "ORDENA EN LINEA",
-    restaurantOptionsOverview: "RESUMEN",
+    restaurantOptionsOverview: "CEDES",
```
Comment (ht1204) [inline src/assets/languages/languageSelector.js:349]: `"SEDES"`

File: src/assets/languages/languageSelector.js [hunk 2]
```diff
@@ -468,20 +471,22 @@ const languageSelector = (lang, str) => {
...
+    userAddressError: "Escribe una dirección valida",
+    userSubtitle: "Aquí puedes ver y editar tus datos",
+    userSaveChanges: "Guardar cambios",
+    signOut: "Cerrar sesión",
+    signOutText: "Si deseas cerrar sesión, haz clic en este botón",
+    generalError: "Ocurrio un error, por favor vuelve a intentarlo.",
+    logInSuccessfull: "¡Has iniciado sesión correctamente!",
+    userUpdatedMessage: "¡Usuario actualizado con éxito!",
+    userUpdateFailedMessage:
+      "No se pudo actualizar el usuario, ocurrio un error, intenta nuevamente.",
```
Comment (ht1204) [inline src/assets/languages/languageSelector.js:485]: `ocurrió`

File: src/assets/languages/languageSelector.js [hunk 3]
```diff
@@ -468,20 +471,22 @@ const languageSelector = (lang, str) => {
...
-    userUpdateFailedMessage: 'No se pudo actualizar el usuario, ocurrio un error, intenta nuevamente.',
-    userPage: 'Ve a la pagina de usuario',
+    restaurantSearchNull:
+      "Ningún restaurante coincide con tu criterios de búsqueda.",
+    userAddressError: "Escribe una dirección valida",
+    userSubtitle: "Aquí puedes ver y editar tus datos",
+    userSaveChanges: "Guardar cambios",
+    signOut: "Cerrar sesión",
+    signOutText: "Si deseas cerrar sesión, haz clic en este botón",
+    generalError: "Ocurrio un error, por favor vuelve a intentarlo.",
```
Comment (ht1204) [inline src/assets/languages/languageSelector.js:481]: `Ocurrió`

**Comment by CamiloGilFranco** [comment]
I'm already working on another branch, could you do the merge and add the changes you request in the following PR

**Comment by CamiloGilFranco** [comment]
I'm already working on another branch, you could do the merge and I'll add the changes you request in the following PR

**Pull request merged**